lib/solr_wrapper/instance.rb

```diff
@@ -103,35 +103,31 @@ def restart
     ##
     # Check the status of a managed Solr service
     def status
```

Comment:
Yea, so it is.  But it could be private and seems to operate as private.  I dunno.  Maybe a major version change?  I like the name `status` for this method.  I started with `status_info` and then `status` did what `started?` does now and basically `started?` can be an alias for `status`.  I could go back to that if this API is important to maintain.